Makefile

```diff
@@ -2,6 +2,9 @@
 APP_ROOT := $(abspath $(lastword $(MAKEFILE_LIST))/..)
 APP_NAME := flyingpigeon
 
+# TODO read in from server configuration
+WPS_URL = http://localhost:8093
+OUTPUT_URL = http://localhost:8093/outputs/
```

Comment:
`OUTPUT_URL` should be `https://pavics.ouranos.ca/wpsoutputs`, exactly like in https://github.com/bird-house/finch/blob/4e9d2baaad0166a76c3e49481934c1eb56af007a/Makefile#L5.  

When running the notebook locally, it already output as `http://localhost:8093/outputs` so there's no point to replace it with the same thing.

We want `https://pavics.ouranos.ca/wpsoutputs` so it matches the production server so the notebook can be used as tutorial notebook.

Reply:
Got it. 
Well, if I understand right, than it is a pavics specific issue, But the tests should be run in general on every server, so better leave the local output. Yes/No?

Reply:
The notebook *should* ideally run against any server, including `localhost` server.  

If run from makefile, it'll be the `localhost` server.  

If run by Jenkins, the hostname `pavics.ouranos.ca` will be replaced by the chosen host (https://github.com/Ouranosinc/PAVICS-e2e-workflow-tests/blob/c3333d3c8de42b3344d3f75df2601f2f4093b11f/runtest#L18-L19).

Recall the `output-sanitize.cfg` include rules to accept both `hostname` and `localhost:port` flavors (https://github.com/Ouranosinc/PAVICS-e2e-workflow-tests/blob/c3333d3c8de42b3344d3f75df2601f2f4093b11f/notebooks/output-sanitize.cfg#L68-L81).